### ISSUE TITLE
UX feedback on search form

### ISSUE THREAD

**Issue by imcbride** [middlebury/midd-frontend#406]
https://support.gmhec.org/TDNext/Apps/49/Tickets/TicketDet.aspx?TicketID=17587

As I continue to use the new Drupal 9 web interface, I have a recommendation for efficiency that I think should be considered.

Using the search box requires several steps:

* click on search box
* move mouse to search field
* type in search phrase

Can this be made more efficient?

I would say that the primary efficiency would be the automatic highlighting of the cursor in the text box upon clicking on the search button. Can the second step be removed and otherwise made to be unneeded?

In a super ideal world, the search box might already be visible, and the cursor also already placed for users to simply start typing, but I realize we're not DuckDuckGo or Google. :-)

In summary, the D9 search interface could save one step versus D7 in an ideal world, but it instead adds one step.
